### PR TITLE
Marksheet: data handling of students

### PR DESCRIPTION
**Pull request by satyamvats5**
We are taking data of students from user and
then we are returning the data requested by the
user.

Signed-off-by: Satyam Kumar<satyamvats5@gmail.com>

### CODE REVIEW

File: pset0/satyamvats5/markssheet/1.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from test import student
+students=[]
```
Comment (RudraNilBasu) [inline pset0/satyamvats5/markssheet/1.py:2]: space between `students` and `[]`

File: pset0/satyamvats5/markssheet/1.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+num=int(input())
```
Comment (RudraNilBasu) [inline pset0/satyamvats5/markssheet/1.py:3]: Here too.

File: pset0/satyamvats5/markssheet/1.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+for i in range(num):
+	val = input()
+	name,roll,m1,m2,m3,m4,m5 = val.split(',')
```
Comment (RudraNilBasu) [inline pset0/satyamvats5/markssheet/1.py:6]: A single space after `,` (Same for next line)

File: pset0/satyamvats5/markssheet/test.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
+class student:
+	def __init__(self,name,roll,m1,m2,m3,m4,m5):
```
Comment (RudraNilBasu) [inline pset0/satyamvats5/markssheet/test.py:2]: Space after `,` and before and after `=` (in the next few lines)

**Review by RudraNilBasu — CHANGES_REQUESTED** [review]
Try not to create a PR from master (will lead to conflicts on sync later)
For coding conventions, Run [PyLint](https://www.pylint.org/) on your code and refer [this](https://google.github.io/styleguide/pyguide.html).

File: pset0/satyamvats5/markssheet/1.py (hunk 3 above)
Comment (0xTheProDev) [reply pset0/satyamvats5/markssheet/1.py:6]: No need to use `val` as temporary. Use `input().split(delim)` instead.

**Comment by 0xTheProDev** [comment]
No need for a test script. Name your files according to the problem name, one for each!

**Pull request closed without merging**